doc/API.md

```diff
@@ -15,14 +15,14 @@ It can be generated running `RAILS_ENV=test rails rswag`.
 
 ## API endpoint documentation
 
-&gt;&gt; [View API documentation](http://petstore.swagger.io/?url=https%3A%2F%2Fraw.githubusercontent.com%2Ffoodcoops%2Ffoodsoft%2Fmaster%2Fdoc%2Fswagger.v1.yml) &lt;&lt;
+&gt;&gt; [View API documentation](http://localhost:3000/api-docs/) &lt;&lt;
 
 The above documentation can communicate with the API directly on a local development
 installation of Foodsoft at [http://localhost:3000/f](http://localhost:3000/f).
 
 You'll need to give access to the application first. This can be done by going to
 _Administration_ > _Configuration_ > _Apps_ in Foodsoft. Select _New Application_,
-enter any name, put `http://petstore.swagger.io/oauth2-redirect.html` in _Redirect URI_
+enter any name, put `http://localhost:3000/api-docs/oauth2-redirect.html` in _Redirect URI_
```

Comment:
Unrelated change, though useful :)

Reply:
ahh couldn't resist, I'll put it in a separate commit.